multicluster/controllers/multicluster/commonarea/clusterinfo_importer_test.go

```diff
@@ -213,22 +219,24 @@ func TestResourceImportReconciler_handleClusterInfo(t *testing.T) {
 
 			if _, err := r.Reconcile(ctx, req); err != nil {
 				t.Errorf("ClusterInfo Importer should handle ResourceImport events successfully but got error = %v", err)
-			} else {
-				gotCIImp := &mcsv1alpha1.ClusterInfoImport{}
-				err := fakeClient.Get(ctx, ciResImportName, gotCIImp)
-				isNotFound := apierrors.IsNotFound(err)
-				if err != nil {
-					if tt.expectedCIImport == nil && !isNotFound {
-						t.Errorf("Expected to get not found error but got error = %v", err)
-					}
-					if tt.expectedCIImport != nil && isNotFound {
-						t.Errorf("Expected to get ClusterInfoImport %v but got not found error = %v", tt.expectedCIImport, err)
-					}
-				} else if tt.expectedCIImport != nil {
-					if !reflect.DeepEqual(tt.expectedCIImport.Spec, gotCIImp.Spec) {
-						t.Errorf("Expected ClusterInfoImport %v but got %v", tt.expectedCIImport.Spec, gotCIImp.Spec)
-					}
+			}
+			gotCIImp := &mcsv1alpha1.ClusterInfoImport{}
+			err := fakeClient.Get(ctx, ciResImportName, gotCIImp)
+			isNotFound := apierrors.IsNotFound(err)
+			if err != nil {
+				if tt.expectedCIImport == nil && !isNotFound {
+					t.Errorf("Expected to get not found error but got error = %v", err)
+				}
+				if tt.expectedCIImport != nil && isNotFound {
+					t.Errorf("Expected to get ClusterInfoImport %v but got not found error = %v", tt.expectedCIImport, err)
 				}
+			} else if tt.expectedCIImport != nil {
+				if !reflect.DeepEqual(tt.expectedCIImport.Spec, gotCIImp.Spec) {
+					t.Errorf("Expected ClusterInfoImport %v but got %v", tt.expectedCIImport.Spec, gotCIImp.Spec)
+				}
+			}
+			if tt.expectedInstalledResImpSize != len(r.installedResImports.List()) {
+				t.Errorf("Unexpected number of installed ResImports after reconciliation")
```

Comment:
better to log what's the expected number and actual number, this may be simpler and contain more information:
```
assert.Equal(t, tt.expectedInstalledResImpSize, len(r.installedResImports.List()), "Unexpected number of installed ResImports after reconciliation")
```

Reply:
Done